R/gsmApp_Server.R

```diff
@@ -123,10 +123,11 @@ gsmApp_Server <- function(
       reactive(input$participant),
       session
     )
-    mod_ParticipantDetails_Server(
+    testing <- mod_ParticipantDetails_Server(
```

Comment:
is this `testing` object intended to be in there, or is it left over from testing/debugging?

Reply:
Oops, yup, it was a leftover from my shinytest2 debugging! Thanks for the catch!